src/Curl/Curl.php

```diff
@@ -96,6 +98,8 @@ public function __construct($base_url = null)
 
         $this->curl = curl_init();
         $this->id = 1;
+        $this->responseParsing = true;
```

Comment:
@Linuxhombre - I implemented deferred parsing of a number of the response components, including the `$response`, but I am not sure I understand what you are suggesting in regards to "custom outbound return messaging/codes and
ALSO variable handling for inbound return strings."

My pull request allows change between returning the parsed response, or if there is an error state.  If you are working off the single `Curl` class, then you pass `false` as the second parameter, and handle the object as you will (if you do not call `$curl->response` your response will not be parsed).

Are you suggesting being able to configure when something will or will not be parsed, down to the error code?